docs/source/configuration_file.rst

```diff
@@ -404,3 +385,102 @@ Available transformations:
+``aleatoric`` can be ``true``.
+
+epistemic
+^^^^^^^^^
+Bool. Model-based uncertainty with `Monte Carlo Dropout <https://arxiv.org/abs/1506.02142>`__.
+
+aleatoric
+^^^^^^^^^
+Bool. Image-based uncertainty with `test-time augmentation <https://doi.org/10.1016/j.neucom.2019.01.103>`__.
+
+n_it
+^^^^
+Integer. Number of Monte Carlo iterations. Set to 0 for no uncertainty computation.
+
+Postprocessing
+--------------
+
+binarize\_prediction
+^^^^^^^^^^^^^^^^^^^^
+Dict. Binarizes predictions according to the given threshold ``thr``. Predictions below the threshold become 0, and
+predictions above or equal to threshold become 1.
+
+- ``thr``: Float. Threshold is between 0 and 1. To use soft predictions
+  (i.e. no binarisation, float between 0 and 1) for metric computation, indicate -1.
+
+fill\_holes
+^^^^^^^^^^^
+Dict. Fill holes in the predictions. No parameters required (e.i., {}).
+
+keep\_largest
+^^^^^^^^^^^^^
+Dict. Keeps only the largest object in prediction. Only nearest neighbors are connected to the center,
+diagonally-connected elements are not considered neighbors. No parameters required (e.i, {})
+
+remove\_noise
+^^^^^^^^^^^^^
+Dict. Sets to zero prediction values under the given threshold ``thr``.
+
+- ``thr``: Float. Threshold is between 0 and 1. Threshold set to ``-1`` will not apply this postprocessing step.
+
+remove\_small
+^^^^^^^^^^^^^
+Dict. Remove small objects from the prediction. An object is defined as a group of connected voxels. Only nearest
+neighbors are connected to the center, diagonally-connected elements are not considered neighbors.
+
+- ``unit``: String. Either "vox" for voxels or "mm3". Indicates the unit used to define the minimal object size.
+- ``thr``: Int. Minimal object size.
+
+threshold\_uncertainty
+^^^^^^^^^^^^^^^^^^^^^^
+Dict. Removes the most uncertain predictions (set to 0) according to a threshold ``thr`` using the uncertainty file with
+the suffix ``suffix``. To apply this method, uncertainty needs to be evaluated on the predictions with the
+:ref:`uncertainty <Uncertainty>` parameter.
+
+- ``thr``: Float. Threshold is between 0 and 1. Threshold set to ``-1`` will not apply this postprocessing step.
+- ``suffix``: String. Indicates the suffix of an uncertainty file. Choices: ``_unc-vox.nii.gz`` for voxel-wise
+  uncertainty, ``_unc-avgUnc.nii.gz`` for structure-wise uncertainty derived from mean value of ``_unc-vox.nii.gz``
+  within a given connected object, or ``_unc-cv`` for structure-wise uncertainty derived from coefficient of variation.
+
+Evaluation parameters
+---------------------
+Dict. Parameters to get object detection metrics (true positive and false detection rates), and this, for defined
+object sizes.
+
+targetSize
+^^^^^^^^^^
+- ``unit``: String. Either "vox" for voxels or "mm3". Indicates the unit used to define the target object sizes.
+- ``thr``: List. containing two int values. The two values will create 3 target size bins: minimal size to first list
```

Comment:
```suggestion
- ``thr``: List. Containing int values. These values will create several consecutive target size bins. For instance with a list of two values, we will have three target size bins: minimal size to first list
```

Reply:
We used it with two values, but in theory the code is code to any values: 4 values --> 5 bins. etc